### PR TITLE
Added install plans to more quickstarts

### PR DESCRIPTION
**Pull request by kidk**
Added a couple of install plans to partner quickstarts, removed a legacy quickstart, and other small clean up.

### CODE REVIEW

**Comment by kidk** [comment]
@jbeveland27 Can you double check my install plans?

File: install/on-host-integration/kafka/install.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: Kafka Monitoring Integration
+description: |
+  The New Relic Kafka on-host integration reports metrics and configuration data from your Kafka service. We instrument all the key elements of your cluster, including brokers (both ZooKeeper and Bootstrap), producers, consumers, and topics.
+
+target:
+  type: on-host-integration
+  destination: host
+
+install:
+  mode: link
```
Comment (polfliet) [inline install/on-host-integration/kafka/install.yml:12]: do we have a targetedInstall for Kafka?

**Review by polfliet — DISMISSED** [review]

File: install/browser/browser-docs/install.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+id: browser-docs
+name: Browser
+title: Browser Monitoring
+description: |
+  With New Relic One's browser monitoring solution, you get full visibility into the complete webpage life cycle of your application or website. Browser measures page load timing, also known as Real User Monitoring (RUM).
+
+target:
+  type: integration
+  destination: cloud
```
Comment (jbeveland27) [inline install/browser/browser-docs/install.yml:9]: Is browser technically an agent? I'd think to make this:

```suggestion
  type: agent
  destination: application
```

Maybe that's wrong though ¯\_(ツ)_/¯

File: install/on-host-integration/kafka/install.yml (hunk 1 above)
Comment (jbeveland27) [reply install/on-host-integration/kafka/install.yml:12]: Unfortunately, no 😕 We set out to have one initially but the setup required a lot of variables set ahead of time, and we never progressed to adding it in.

**Review by jbeveland27 — COMMENTED** [review]
Left a comment

**Review by jbeveland27 — APPROVED** [review]

**Review by polfliet — APPROVED** [review]

**Pull request merged**